app/src/main/java/com/example/playlistmaker2/PlayerDisplayActivity.kt

```diff
@@ -19,15 +21,29 @@ class PlayerDisplayActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityPlayerDisplayBinding
 
-    private lateinit var arrowBack : ImageButton
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+        private const val REFRESH_MILLIS = 500L
+    }
+
+    private var playerState = STATE_DEFAULT
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Сам объект не должен пересоздаваться, лучше пользоваться одним и тем же объектом плеера, переиспользуя его по максимуму